assets/airports/vecc.json

```diff
@@ -0,0 +1,606 @@
+			["KAPOL", "CC107", "ONOTO*"],
+			["KAPOL", "CC105", "SAREK*"]
+		]
+	},
+	"KESAR": {
+		"icao": "KESAR",
+		"name": "KESAR",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"BOKEV": ["LEGOS", "CC110", "BOKEV"],
+			"KAKID": ["LEGOS", "KAKID"],
+			"SURUP": ["LEGOS", "KAKID", "SURUP"]
+		},	
+		"body": ["KESAR"],
+		"draw": [
+			["_01R", "KESAR"], 
+			["_01L", "KESAR*"],
+			["KESAR", "LEGOS", "KAKID*", "SURUP*"]
+		]
+	},
+	"RIKLA": {
+		"icao": "RIKLA",
+		"name": "RIKLA",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"JJS":		["CC101", "JJS"],
+			"TEPAL": 	["CC201", "CC203", "CC103", "TEPAL"]
+		},	
+		"body": ["RIKLA"],
+		"draw": [
+			["_01R", "RIKLA"], 
+			["_01L", "RIKLA*"],
+			["RIKLA", "CC101", "JJS*"],
+			["RIKLA", "CC201", "CC203", "CC103", "TEPAL*"]
+		]
+	},
+	"SEGOL": {
+		"icao": "SEGOL",
+		"name": "SEGOL",
+		"rwy": {
+			"19L": [],
+			"19R": []
+		},
+		"transitions": {
+			"BEMAK": ["CC312", "BEMAK"],
+			"NOKAT": ["CC312", "NOKAT"],
+			"SUMAG": ["SUMAG"]
+		},	
+		"body": ["SEGOL"],
+		"draw": [
+			["_01R", "SEGOL"], 
+			["_01L", "SEGOL*"],
+			["SEGOL", "CC312", "BEMAK*"],
+			["SEGOL", "CC312", "NOKAT*"],
+			["SEGOL", "SUMAG*"]
+		]
+	}
+  },
+ 
+  "departures": {
+    "airlines": [
+		["xax",  10],
+		["aic", 190],
+		["llr",  30],
+		["uae",  10],
+		["etd",  10],
+		["gow",  40],
+		["igo", 290],
+		["jai", 160],
+		["sej", 120],
+		["ual",  20],
+		["vti",  10],
+		["ces",  10],
+		["hda",  10],
+		["slk",  10],
+		["cpa",  20]
+    ],
+    "destinations": [
+		"GUMAS", "KAPOL", "KESAR", "RIKLA", "SEGOL", "BAMUR", "BASAT", "LOPEM", "NIBLU", "PITAM"
+    ],
+    "type": "random",
+    "offset": 0,
+    "frequency": 20
+  },
+  "arrivals": [
+	{	
+		"type": 		"cyclic",
+		"fixes":		["CC112", "AVPOP", "CC210", "CC304"],
+		"frequency": 	4,
+		"altitude":		12000,
+		"speed":		330,
+		"offset":		2,
+		"airlines": [
+			["cpa", 2],
+			["xax", 3],
+			["aic", 15],
+			["llr", 5],
+			["ual", 3],
+			["gow", 3],
+			["igo", 20],
+			["jai", 10],
+			["sej", 8],
+			["ces", 3],
+			["hda", 3],
+			["slk", 3],
+			["cpa", 5]
+		]
+	},
+	{	
+		"type": 		"cyclic",
+		"fixes":		["CC102", "BAMUR", "CC321", "CC303", "CC316"],
+		"frequency": 	4,
+		"altitude":		12000,
+		"speed":		330,
+		"offset":		2,
+		"airlines": [
+			["aic", 15],
+			["llr", 5],
+			["ual", 5],
+			["gow", 3],
+			["igo", 20],
+			["jai", 10],
+			["sej", 8],
+			["uae", 5],
+			["etd", 5]
+		]
+	},
+	{	
+		"type": 		"cyclic",
+		"fixes":		["BEMAK", "DIXON", "CC330", "DINAP", "CC329"],
+		"frequency": 	4,
+		"altitude":		12000,
+		"speed":		330,
+		"offset":		2,
+		"airlines": [
+			["cpa", 2],
+			["xax", 3],
+			["aic", 15],
+			["llr", 5],
+			["ual", 3],
+			["gow", 3],
+			["igo", 20],
+			["jai", 10],
+			["sej", 8],
+			["ces", 3],
+			["hda", 3],
+			["slk", 3],
+			["cpa", 5]
+		]
+	},
+	{	
+		"type": 		"cyclic",
+		"fixes":		["NOKAT", "DIKAK", "CC302", "TONAP"],
```

Comment:
This fix, "TONAP" is not defined above in `fixes`, and aircraft spawning on this arrival stream are eventually causing the game to crash because it finds out that it doesn't know where to send the aircraft. Could you add this fix please?

Other than this, I believe everything is working properly!